mrp_sale_info/models/stock_move.py

```diff
@@ -11,6 +11,25 @@ def _get_orig_created_production_ids(self):
         if self.created_production_id:
             return self.created_production_id
         mrp_production_ids = set()
-        for move in self.move_orig_ids:
-            mrp_production_ids.update(move._get_orig_created_production_ids().ids)
+        # There's a scenario where the current move is in self.move_orig_ids.move_orig_ids,
```

Comment:
It is probably a data issue here, but in any case we need to be protected against infinite loop in the implementation.